### PR TITLE
GW-40 develop activities section to be responsive

### PR DESCRIPTION
**Pull request by chrishadrian**
- Added mobile responsiveness for activites section
- Renewed activities images
- Refactored some styling

Deploy preview: https://deploy-preview-18--gisaubcv3.netlify.app/

### CODE REVIEW

**Review by Julian-UBC — CHANGES_REQUESTED** [review]
- [x] Can you rebase the branch with develop first?

**Review by jenaarianto — APPROVED** [review]

File: src/pages/home/components/About.js [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
 import about_image from "assets/about/about.png";
-import about_mobile from "assets/about/about_mobile.svg";
+import about_mobile from "assets/about/about_mobile.png";
 import sunshine_image from "assets/sunshine.svg";
 import { useNavigate } from "react-router-dom";
 import { Button } from "shared/components/Button/Button";
 import { useMediaQuery } from "react-responsive";
 
-const typographyStyle = "text-[12px] sm:text-[15px] xl:text-[20px] leading-tight font-proxima-nova my-5 w-full sm:pr-6";
+const typographyStyle = "text-base sm:text-base xl:text-xl font-proxima-nova my-5 w-full sm:pr-6";
```
Comment (Julian-UBC) [inline src/pages/home/components/About.js:10]:
```suggestion
const typographyStyle = "text-base xl:text-xl font-proxima-nova my-5 w-full sm:pr-6";
```

**Review by Julian-UBC — CHANGES_REQUESTED** [review]

**Review by Julian-UBC — COMMENTED** [review]
Good job @chrishadrian!! It looks good

**Review by Julian-UBC — COMMENTED** [review]
Resolved

**Review by Julian-UBC — APPROVED** [review]
Approved

**Pull request merged**